### ISSUE TITLE
1.9.x - Update metaphors templates to support github

### ISSUE THREAD

**Issue by 6za** [kubefirst/kubefirst#332]
# Overview 

Due #331 and #330, we have to allow [this line](https://github.com/kubefirst/metaphor-template/blob/main/.argo/npm-run.yaml#L21) to be tokenized, to allow in gitlabm it points to `git-checkout-with-gitops`, but for github it would point to a new cwft `git-checkout-with-gitops-ssh`.

`git-checkout-with-gitops-ssh` would be used clone ssh repos. This would allow smaller impact on the metaphors app, keeping its general structure, but allow to clone properly the repo. 

```yaml
    - - name: checkout
        templateRef:
          name: cwft-git
          template: <CHECKOUT_CWFT_TEMPLATE>
          clusterScope: true 
        arguments:
          parameters:
          - name: appDir
            value: "{{workflow.parameters.appDir}}"
          - name: branch
            value: "{{workflow.parameters.branch}}"
          - name: gitRepoUrl
            value: "{{workflow.parameters.gitRepoUrl}}"
```

**Comment by 6za** [comment]
Waiting test of github

**Comment by 6za** [comment]
To be addressed on #340
